### ISSUE TITLE
FixedUpdate synchronization do not works in 2020.2

### ISSUE THREAD

**Issue by longbombus** [Cysharp/UniTask#208]
Steps to reproduce:
1. Empty project from Unity 2020.2.1f1
2. UniTask via git package with url https://github.com/Cysharp/UniTask.git?path=src/UniTask/Assets/Plugins/UniTask#2.0.37
3. Fixed time step set to 1/16 = 0.0625
4. Async method code:
```
private async UniTask MethodAsync(CancellationToken cancellationToken)
{
	while (true)
	{
		Debug.Log($"Frame={Time.frameCount} FixedTime={Time.fixedTime}");
		await UniTask.WaitForFixedUpdate(cancellationToken);
	}
}
```

log result:
```Frame=0 FixedTime=0
Frame=1 FixedTime=0
Frame=2 FixedTime=0
Frame=3 FixedTime=0
Frame=4 FixedTime=0,125
Frame=5 FixedTime=0,125
Frame=6 FixedTime=0,125
Frame=7 FixedTime=0,1875
Frame=8 FixedTime=0,1875
...
```

looks like it yields only for one frame update, not fixed update.

[Sample project](https://drive.google.com/file/d/1Q9NJ-Iv7tzGCxCkBuxy1TizUhGCdcLf_/view)

**Comment by WallyCZ** [comment]
I have exactly the same problem (Unity 2020.2), but it seems that it is only wrong in the Editor. When I try to build some platform player, it seems, that it is working correctly...

**Comment by neuecc** [comment]
Thank you for your valuable report.
This was a very serious bug.
I have just released 2.1.0, which includes a fix for 2020.2.
